.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Description of change
+
+_Please provide a description of change here._
+
+### Pull Request check-list
+
+_Please make sure to review and check all of these items:_
+
+- [ ] Does `make -j8 test` (UNIX) or `vcbuild test nosign` (Windows) pass with
+  this change (including linting)?
+- [ ] Is the commit message formatted according to [CONTRIBUTING.md][0]
+- [ ] Is a regression test (or benchmark if this is a performance-related
```

Comment:
It might make it a little more wordy, but what about instead something like:

<pre>
If this PR modifies runtime code (e.g. in `lib/` or `src/`), is a regression
test included (or a benchmark in case of a performance-related change)?
</pre>

I'm not sure if that's the best terminology to identify that type of code, but the point was to differentiate between say code in `test/`, `tools/`, or elsewhere and code that gets executed during runtime.